pkg/advancedapi/advancedApi_test.go

```diff
@@ -66,22 +66,22 @@ func Test_cannot_get_delegation_proof(t *testing.T) {
 	err = advancedapi.RegisterUpdatedDocument(ctx, resolver, subjectDoc, test.ValidKeyPairPlop, subjectIssuer)
 	assert.NilError(t, err)
 
-	// Create and add second key to document so original key can be removed
+	// Create and add second key to document so original key can be revoked
 	secondKey, err := crypto.GetPrivateKeyFromExponent("baddbaddbaddbaddbaddbaddbaddbaddbaddbaddbaddbaddbaddbaddbaddbadd")
 	assert.NilError(t, err)
 	_, secondKeyPublicBase58, err := crypto.GetPublicKeysFromPrivateKey(secondKey)
 	assert.NilError(t, err)
 	err = advancedapi.AddPublicKeyToDocument(ctx, resolver, subjectDoc, "#second", secondKeyPublicBase58, subjectIdentity)
 	assert.NilError(t, err)
 
-	err = advancedapi.RemovePublicKeyFromDocument(ctx, resolver, nil, "#name", subjectIdentity)
+	err = advancedapi.RevokePublicKeyFromDocument(ctx, resolver, nil, "#name", subjectIdentity)
 	assert.NilError(t, err)
 
 	subjectDoc, err = resolver.GetDocument(ctx, subjectDoc.ID)
 	assert.NilError(t, err)
 
 	_, _, err = advancedapi.CreateDelegationProof(delegatingIssuer, subjectDoc, test.ValidKeyPairPlop)
-	assert.ErrorContains(t, err, "unable to find public key in document matching key pair secrets")
+	assert.NilError(t, err)
```

Comment:
should this be the case?

Reply:
@StephanieBracaloni input on this test case? Should a delegation with a revoked key still be able to create a proof?